package.json

```diff
@@ -7,7 +7,18 @@
     "start": "begin dev",
     "lint": "eslint ./app/**/*.mjs --fix"
   },
+  "engines": {
+    "node": ">=18"
+  },
   "devDependencies": {
+    "@architect/plugin-lambda-invoker": "^1.0.0",
+    "@aws-sdk/client-apigatewaymanagementapi": "^3.303.0",
+    "@aws-sdk/client-dynamodb": "^3.303.0",
+    "@aws-sdk/client-s3": "^3.305.0",
+    "@aws-sdk/client-sns": "^3.303.0",
+    "@aws-sdk/client-sqs": "^3.303.0",
+    "@aws-sdk/client-ssm": "^3.303.0",
+    "@aws-sdk/lib-dynamodb": "^3.303.0",
```

Comment:
this list would scare me as a user; since it's a dev dep, can we just add @architect/architect or sandbox?

Reply:
More on this soon.